helm/templates/gateway/_gateway-deployment-probes.tpl

```diff
@@ -0,0 +1,107 @@
+{{- define "gateway.httpHeadersProbes" -}}
+    {{- $httpHeadersProbes := dict }}
+    {{- if and
+        .Values.gateway.services.core.http.enabled
+        (eq (default "basic" .Values.gateway.services.core.http.authentication.type) "basic")
+        .Values.gateway.services.core.http.authentication.password
+    }}
+      {{- $httpHeadersProbes = dict "httpHeaders" (list (dict
+          "name" "Authorization"
+          "value" (printf "Basic %s" (printf "admin:%s" .Values.gateway.services.core.http.authentication.password | b64enc))
+      )) }}
+    {{- end }}
+
+    {{- $httpHeadersProbes | toYaml }}
+{{- end }}
+
+{{- define "gateway.gatewayServiceCoreScheme" -}}
+    {{- $gatewayServiceCoreScheme := ( ternary "HTTPS" "HTTP" .Values.gateway.services.core.http.secured) }}
+    {{- $gatewayServiceCoreScheme }}
+{{- end }}
+
+{{- define "gateway.computeLivenessProbe" -}}
+    {{- $httpHeadersProbes := (include "gateway.httpHeadersProbes" . | fromYaml) -}}
+    {{- $gatewayServiceCoreScheme := (include "gateway.gatewayServiceCoreScheme" .) -}}
+
+    {{- $defaultLivenessProbe := dict
+        "initialDelaySeconds" 30
+        "periodSeconds" 30
+        "timeoutSeconds" 3
+        "successThreshold" 1
+        "failureThreshold" 3
+        "httpGet" (merge (dict
+          "path" "/_node/health?probes=http-server"
+          "scheme" $gatewayServiceCoreScheme
+          "port" (.Values.gateway.services.core.http.port | default 18082)
```

Comment:
**issue**: does this default port is the right one ? 18082 is usually the one for APIM. AM ius using 18092 or 18093 according to the service (cf removed default values from the values.yaml)